oak_functions/loader/Cargo.toml

```diff
@@ -6,8 +6,8 @@ edition = "2018"
 license = "Apache-2.0"
 
 [features]
-# Feature that allows logging of potentially sensitive content.
-# Should only be used for debugging purposes.
+# Feature that allows logging of potentially sensitive content, or using experimental features.
+# Should only be used for debugging purposes or enabling experimental features.
```

Comment:
imo we can probs remove at that line at this point :)

Reply:
Do you mean that the entire comment about this feature can be removed?